headers/data/overlay29.h

```diff
@@ -77,6 +77,7 @@ extern struct fx64_16 DAMAGE_FORMULA_MAX_BASE;
 extern struct fx64_16 WONDER_GUARD_MULTIPLIER;
 extern struct fx64_16 DAMAGE_FORMULA_MIN_BASE;
 extern struct damage_negating_exclusive_eff_entry TYPE_DAMAGE_NEGATING_EXCLUSIVE_ITEM_EFFECTS[28];
+extern uint8_t TWO_TURN_STATUSES[11];
```

Comment:
Should this be `struct status_id_8`?

Reply:
No, it's not the combined `status_id` enum. It's compared to the value of [monster.statuses.bide](https://github.com/UsernameFodder/pmdsky-debug/blob/master/headers/types/dungeon_mode/dungeon_mode.h#L132), which is a regular `uint8_t`. The value matches the enum [status_two_turn_id](https://github.com/UsernameFodder/pmdsky-debug/blob/master/headers/types/dungeon_mode/enums.h#L276), except that this enum is 16-bit while `TWO_TURN_STATUSES` has 8-bit values.

Reply:
Let's make an 8-bit version of the enum and use that.

Reply:
Since the other status classes are handled similarly, I made enums for all those as well as this.